nautobot/core/api/views.py

```diff
@@ -959,4 +958,17 @@ def get(self, request):
             model_form_instance = TempForm(auto_id="id_for_%s")
 
         bound_field = form_field.get_bound_field(model_form_instance, field_name)
-        return Response({"dom_element": bound_field.as_widget()})
+        if "as_json" in request.GET:
```

Comment:
Does jQuery's request provide an `Accept` header? If it's asking for `application/json` then I'd like to suggest a potential refactor here:

- The legacy UI returns just plain HTML (ie `Response(bound_field.as_widget())`) and jQuery passes the `Accept: */*` or `Accept: text/html` header
- The new UI requests `Accept: application/json` and you get the data below